documentation/gprof.md

```diff
@@ -0,0 +1,58 @@
+# Profiling user Trusted Applications with `gprof`
+
+The configuration option `CFG_TA_GPROF_SUPPORT=y` enables OP-TEE to
+collect profiling information from Trusted Applications running in user
+mode and compiled with `-pg`.
+Once collected, the profiling data are formated in the `gmon.out` format
+and sent to `tee-supplicant` via RPC, so they can be saved to disk and
+later processed and displayed by the standard `gprof` tool.
+
+## Usage
+
+- Build OP-TEE OS with `CFG_TA_GPROF_SUPPORT=y`. You will likely need to
+  increase the TEE heap: for instance `CFG_CORE_HEAP_SIZE=262144`. You
```

Comment:
I've used this value quite some time ago when testing on D02, but the memory allocation was a bit different than the final version.
I'll double check this and give more precise information (i.e., the additional amount of memory that is required as a function of the TA `.text` size).